setup_payram.sh

```
@@ -2063,30 +2077,197 @@ reset_payram_environment() {
+  print_color "gray" "  • Data directory: $PAYRAM_CORE_DIR"
+  [[ -d "$PAYRAM_CORE_DIR" ]] && print_color "gray" "    └─ Status: ✅ Exists ($(du -sh "$PAYRAM_CORE_DIR" 2>/dev/null | cut -f1) - will remove)" || print_color "gray" "    └─ Status: ❌ Not found"
+  if [[ -d "$PAYRAM_CORE_DIR" ]]; then
+    [[ -d "$PAYRAM_CORE_DIR/db" ]] && print_color "gray" "    └─ Database: ✅ Found (will remove)" || print_color "gray" "    └─ Database: ❌ Not found"
+    [[ -d "$PAYRAM_CORE_DIR/logs" ]] && print_color "gray" "    └─ Logs: ✅ Found (will remove)" || print_color "gray" "    └─ Logs: ❌ Not found"
+  fi
+  echo
+  
+  print_color "yellow" "🔒 SSL/TLS Components:"
+  print_color "gray" "  • Let's Encrypt certificates: /etc/letsencrypt/"
+  if [[ -d "/etc/letsencrypt" ]] && [[ "$(find /etc/letsencrypt -name "*.pem" 2>/dev/null | wc -l)" -gt 0 ]]; then
+    local cert_count=$(find /etc/letsencrypt -name "*.pem" 2>/dev/null | wc -l)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)